api.planx.uk/modules/flows/createFlow/controller.ts

```diff
@@ -0,0 +1,48 @@
+import { z } from "zod";
+import type { ValidatedRequestHandler } from "../../../shared/middleware/validate.js";
+import { ServerError } from "../../../errors/index.js";
+import { createFlow } from "../../../helpers.js";
+import type { Flow } from "../../../types.js";
+
+interface CreateFlowResponse {
+  message: string;
+  id: Flow["id"];
+  slug: Flow["slug"];
+}
+
+export const createFlowSchema = z.object({
+  body: z.object({
+    teamId: z.number(),
+    slug: z.string(),
+    name: z.string(),
+    data: z.any(), // FlowGraph
```

Comment:
Was a little stumped here and landed on `z.any()`, is there a way for `zod` to access an externally defined type instead or a generic "json" type? Repeating planx-core's `FlowGraph` type as a `z.object({})` directly felt like the wrong direction?

Reply:
Ah of course, there's actually no good reason that `data` needs to be passed from the frontend anyways here ! Have updated to simply define `const initialFlowData: FlowGraph = {...}` API-side which negates my zod issue :slightly_smiling_face: